scenarios/bring-your-own-data/bringYourOwnData.md

```diff
@@ -0,0 +1,140 @@
+# Use Azure OpenAI On Your Data With openai-node Library
+
+## Objective
+
+Use openai-node Azure client to generate answers from your data using Azure On Your Data feature.
+
+This tutorial uses the following services:
+
+- Access to Azure OpenAI Service - you can apply for access [here](https://go.microsoft.com/fwlink/?linkid=2222006)
+- Access to Azure Search Service 
+
+## Time
+
+You should expect to spend 5-10 minutes running this sample. 
+
+## Before you begin
+
+### Set Environment Variables
+
+To start with let us create a config `.env` file with your project details. This file can be used in this sample or other samples to connect to your workspace. 
+
+```ts
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
```suggestion
import 'dotenv/config'
```

Reply:
Also, I just noticed that the language listed is JavaScript but the code is typed as TypeScript, I think we should make sure it is proper JS code.